tests/phpunit/Writer/MSSQLTest.php

```diff
@@ -313,20 +326,14 @@ public function testGetTableInfo()
         }
     }
 
-    public function testValidateTable()
-    {
-        $tables = $this->config['parameters']['tables'];
-        $table = $tables[0];
-        $this->writer->create($table);
-        $this->writer->validateTable($table);
-    }
-
+    /**
+     * @throws \Keboola\DbWriter\Exception\UserException
```

Comment:
To samé tady

Reply:
Ja som si to nevymyslel, "radi" mi to PHPStorm, inak tam dava warning